src/llmcompressor/modifiers/transform/spinquant/base.py

```diff
@@ -262,7 +262,7 @@ def _create_r4_scheme(self) -> TransformScheme:
             randomize=self.randomize,
             requires_grad=self.learnable,
             precision=self.precision,
-            block_size=self.transform_block_size,
+            head_dim=self.transform_block_size,
```

Comment:
I assume this is for backwards compatibility?

Reply:
right, to revert the deprecation and just go back to using `head_dim`